src/py/flwr/client/numpy_client.py

```diff
@@ -86,6 +89,17 @@ def get_parameters(self) -> List[np.ndarray]:
             The local model parameters as a list of NumPy ndarrays.
         """
 
+    # @abstractmethod
+    def get_properties(self, ins: PropertiesIns) -> PropertiesRes:
```

Comment:
The argument should be called `config` and the type should be the same as in `fit/evaluate`

Reply:
Some background: `Ins/Res` types are only used in `Client` whereas `NumPyClient` uses higher-level / more Pythonic / easier-to-use types

Reply:
As in `Ins/Res` use gRPC messages, and `NumpyClient` uses dictionary?